src/messages.js

```diff
@@ -57,4 +57,8 @@ export default {
   'vocab.taxon.local.collectionName': 'PAHMA Taxonomic Names',
   'option.taxonomicStatuses.valid': 'current',
   'option.taxonomicStatuses.invalid': 'obsolete',
+
+  // valuation
+
+  'field.valuationcontrols_common.valuationcontrolRefNumber.name': 'Valuation control reference number',
```

Comment:
Don't need this. In 4.1, look in tenants/pahma/bundle/core-messages.properties-overlay to see what messages were customized for pahma. This message wasn't customized.